qt/python/mantidqt/project/recovery/recoverygui/test/test_projectrecoverypresenter.py

```diff
@@ -0,0 +1,220 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+
+import sys
+import unittest
+
+from mantidqt.project.recovery.recoverygui.projectrecoverypresenter import ProjectRecoveryPresenter
+
+if sys.version_info.major >= 3:
+    from unittest import mock
+else:
+    import mock
+
+
+def raise_exception():
+    raise Exception
+
+
+class ProjectRecoveryPresenterTest(unittest.TestCase):
+    def setUp(self):
+        self.prp = ProjectRecoveryPresenter(mock.MagicMock(), model=mock.MagicMock())
```

Comment:
The mocks should be moved into member variables, and have assertions in the tests

Reply:
Both of those are accessible inside of the object, as member variables which are asserted on.